src/Tools/ExternalAccess/OmniSharp/Analyzers/OmnisharpAnalyzerLoaderFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Analyzers
+{
+    internal static class OmnisharpAnalyzerAssemblyLoaderFactory
+    {
+        public static IAnalyzerAssemblyLoader CreateShadowCopyAssemblyLoader(string? baseDirectory = null)
```

Comment:
Right now Omnisharp is using a copy/paste of our shadow copy implementation. It hasn't gotten some of the fixes we've done over the years; this is simply going to offer them the opportunity to use ours directly.

Reply:
We should also just consider making our implementation public entirely.